lib/widgets/about_app.dart

```diff
@@ -0,0 +1,241 @@
+                child: Column(
+                  children: <Widget>[_appContactAndGit],
+                ),
+              )
+            ],
+          ),
+          Row(
+            children: [
+              TextButton(
+                  onPressed: () {
+                    _launchGitHub();
+                  },
+                  child: Row(
+                    children: [
+                      Text(
+                        "GitHub Repository ",
+                        style:
+                            TextStyle(color: Colors.lightGreen, fontSize: 14.0),
+                      ),
+                      Icon(
+                        FontAwesomeIcons.github,
+                        color: Colors.lightGreen,
+                      )
+                    ],
+                  ))
+            ],
+            mainAxisAlignment: MainAxisAlignment.center,
+          ),
+          Row(
+            children: [
+              TextButton(
+                  onPressed: () {
+                    _launchMailTo();
+                  },
+                  child: Row(
+                    children: [
+                      Text(
+                        'E-Mail ',
+                        style:
+                            TextStyle(color: Colors.lightGreen, fontSize: 14.0),
+                      ),
+                      Icon(
+                        Icons.mail_rounded,
+                        color: Colors.lightGreen,
+                      )
+                    ],
+                  ))
+            ],
+            mainAxisAlignment: MainAxisAlignment.center,
+          ),
+          Row(
+            children: [
+              TextButton(
+                  onPressed: () {
+                    _launchLicense();
+                  },
+                  child: Row(
+                    children: [
+                      Text(
+                        'Lizenz ',
+                        style:
+                            TextStyle(color: Colors.lightGreen, fontSize: 14.0),
+                      ),
+                      Icon(
+                        Icons.book,
+                        color: Colors.lightGreen,
+                      )
+                    ],
+                  ))
+            ],
+            mainAxisAlignment: MainAxisAlignment.center,
+          ),
+          Row(
+            children: [
+              TextButton(
+                onPressed: () {
+                  Navigator.push(
+                    context,
+                    MaterialPageRoute(
+                        builder: (context) =>
+                            Privacy()), // We use the Navigator to Route to the settings page wich is located in a new .dart file
+                  );
+                },
+                child: Text(
+                  'Datenschutz ',
+                  style: TextStyle(color: Colors.lightGreen, fontSize: 14.0),
+                ),
+              )
+            ],
+            mainAxisAlignment: MainAxisAlignment.center,
```

Comment:
Maybe think about putting all the rows with the TextButtons in a class/function. You can then pass the appropriate color/icon/etc to the class/function